README.md

```diff
@@ -60,52 +60,54 @@ RoR FOOD RECIPES is a Ruby on Rails application that keeps track of all your foo
 
 ### Key Features <a name="key-features"></a>
 
-There are few striking features of this project, namely:
+> Describe between 1-3 key features of the application.
 
-- Create, show or delete a food
-- Create, display or delete a food recipe
-- Display a shopping list
+- Create a category for your transactions
+- Add a transaction based on a preffered category
+- Display the details of all transactions in a particular category
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!--
-## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](https://rails-3xzd.onrender.com)
```

Comment:
- [ ] When running your project locally everything Is good 🎉  but the live link Is showing this 👇 


![image](https://user-images.githubusercontent.com/75126481/224351848-11afae79-56a2-4e20-8964-1f7526fa0027.png)